webpack/webpack.prod.cjs

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const CssMinimizerPlugin = require('css-minimizer-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const { mergeWithRules } = require('webpack-merge');
+
+const generateConfig = require('./webpack.config.cjs');
+const constants = require('./constants.cjs');
+
+const baseConfig = generateConfig(constants);
+
+const optimizationOverride = {
```

Comment:
Why is the distributed release optimized but not gh-pages?

Reply:
Oh, yes. It should be optimized for gh-pages too.